### ISSUE TITLE
[BUG] Ran into unexpected error when getting logs for waagent-network-setup.service service.

### ISSUE THREAD

**Issue by taoyama** [Azure/WALinuxAgent#2228]
If system locale is other than English (ex. Japanese), ascii encoding fails because [command result of journalctl](https://github.com/Azure/WALinuxAgent/blob/f6a38c7403a6e40aef1ed183907e55ca345d6ffe/azurelinuxagent/common/persist_firewall_rules.py#L246) contains multibyte characters.

**repro steps**
1.  Create VM
```bash
$ az group create -g myResourceGroup -l japaneast
$ az vm create -g myResourceGroup -n rhel79 --image RedHat:RHEL:7_9:7.9.2020111301
```
2. SSH to the VM
3. Stop firewall and enable waagent-network-setup.service
```bash
[root@rhel79 ~]# systemctl stop firewalld
[root@rhel79 ~]# systemctl disable firewalld
[root@rhel79 ~]# iptables -F -t security
[root@rhel79 ~]# localectl set-locale LANG=ja_JP.UTF8
[root@rhel79 ~]# systemctl restart waagent
[root@rhel79 ~]# systemctl start waagent-network-setup.service
[root@rhel79 ~]# systemctl restart waagent
```
4.  Logging failed due to encoding error
```bash
[root@rhel79 ~]# grep ascii /var/log/waagent.log
2021-04-22T14:23:04.842568Z WARNING ExtHandler ExtHandler Ran into unexpected error when getting logs for waagent-network-setup.service service. Error: 'ascii' codec can't encode character u'\u6728' in position 17: ordinal not in range(128)
```

**Distro and WALinuxAgent details (please complete the following information):**
 - Distro and Version: RHEL7
 - WALinuxAgent version: From 2.2.54

**Comment by larohra** [comment]
Thanks for reporting this, I've sent out a PR for the fix for this - #2233

**Comment by larohra** [comment]
Fixed in #2233
